session.go

```diff
@@ -122,7 +126,7 @@ func (session *Session) IsClosed() bool {
 func (session *Session) GetBytesRead() uint64 {
 	session.RLock()
 	defer session.RUnlock()
-	return session.bytesRead
+	return session.bytesReceived
```

Comment:
Create a new function to return session.bytesReceived

Reply:
bytesReceived is removed